doc/pmon/sonic_thermal_control_test_plan.md

```diff
@@ -0,0 +1,250 @@
+# Thermal Control Test Plan
+
+## Table of Content
+
+- [Overview](#overview)
+- [Scope](#scope)
+- [Test Structure](#test-structure)
+  - [Setup Configuration](#setup-configuration)
+  - [Ansible and Pytest](#ansible-and-pytest)
+- [Test Cases](#test-cases)
+  - [Show FAN Status Test](#show-fan-status-test)
+  - [Show Thermal Status Test](#show-thermal-status-test)
+  - [FAN Test](#fan-test)
+  - [PSU Absence Test](#psu-absence-test)
+  - [Invalid Policy Format Load Test](#invalid-policy-format-load-test)
+  - [Invalid Policy Value Load Test](#invalid-policy-value-load-test)
+
+
+## Overview
+
+The purpose is to test functionality of thermal control feature on the SONiC switch DUT. The thermal control feature contains 3 major functions: FAN status monitor, thermal status monitor and thermal control policy management.
+
+- FAN status monitor read FAN status via platform API every 60 seconds and save it to redis database. User can fetch FAN status via command line `show platform fanstatus`.
+- Thermal status monitor read thermal status via platform API every 60 seconds and save it to redis database. User can fetch thermal status via command line `show platform temperature`.
+- The thermal control policy is defined in a JSON file and loaded by thermal control daemon in pmon docker. Thermal control daemon collects thermal information and matches thermal conditions. Once some thermal conditions match, related thermal actions will be triggered. 
+
+More detail on thermal control feature can be found in this [document](https://github.com/keboliu/SONiC/blob/thermal_control_design/thermal-control-design.md).
```

Comment:
Suggest to rephrase:
A more detailed design and function description can be found it ...'